balance/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+#[macro_use]
+extern crate futures;
+extern crate http;
+extern crate h2;
+extern crate tower_balance;
+extern crate tower_h2;
+
+use futures::Poll;
+use tower_balance::load::Measure;
+use tower_h2::Body;
+
+pub struct MeasureFirstData;
+
+pub struct MeasureEos;
+
+pub struct MeasureFirstDataBody<T, B> {
```

Comment:
Since `MeasureFirstBody` and `MeasureFirstEosBody` wrap and replace the request's body type, it would be nice if they could derive or otherwise implement `Debug` when the wrapped body is `Debug`.

Reply:
I think there are probably other traits that body type parameters are also commonly bounded with. It might be useful for these types to implement those traits as well when `B` does, to avoid breaking code that bounds the body type when adding a `Measure`?